src/app/app-component.service.ts

```diff
@@ -0,0 +1,189 @@
+import { Injectable } from '@angular/core';
+import { weatherData, widgetListInterface, widgetWeatherData } from './interfaces';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AppComponentService {
+
+  constructor() { }
+
+  weatherDataTransformation(inputData: weatherData): widgetWeatherData[] {
+
+    let limitDays = 5; //НАСТРОЙКА КОЛЛИЧЕСТВА ДНЕЙ ДЛЯ ПОКАЗА
```

Comment:
Please use English comment, you are a developer and one of the main developers languages is English, so please use English instead of Russian.